src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,23 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json"];
+const IOS_INDICATORS = ["info.plist", "project.pbxproj", "podfile", "package.swift"];
+const ANDROID_INDICATORS = ["androidmanifest.xml", "build.gradle"];
+
+// given a directory, determine the platform type
+export async function getPlatformFromFolder(dirPath: string) {
```

Comment:
Would love a unit test for this - you can stub out fs.readdir, and then add tests for a few different file lists. Right now these are pretty simple, but it will be nice to have tests to prevent accidental breaks in the future.

Reply:
are you ok with mock-fs dependency?

Reply:
Yeah, that seems safe to add (and it'll be useful for me in unit testing the init refactor)